erpnext/hr/doctype/daily_work_summary_group/daily_work_summary_group.py

```diff
@@ -0,0 +1,59 @@
+# # -*- coding: utf-8 -*-
+# # Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and contributors
+# # For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.model.document import Document
+import frappe.utils
+from frappe import _
+
+
+class DailyWorkSummaryGroup(Document):
+	def validate(self):
+		if self.users:
+			if not frappe.flags.in_test and not is_incoming_account_enabled():
+				frappe.throw(_('There must be a default incoming Email Account enabled for this to work. '
```

Comment:
translatable strings must be in single line. Use smaller strings

Maybe just _('Please enable default incoming account before creating Daily Work Summary Group')

Reply:
Fixed https://github.com/frappe/erpnext/pull/12944/commits/343192078b2fffb4a4b524731713d422e7f9dd89